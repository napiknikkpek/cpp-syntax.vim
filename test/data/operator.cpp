void operator+(A, A a, int, int a);
void A::operator+(A, A a, int, int a);
void operator()(A, A a, int, int a);
void operator+=(A, A a, int, int a);
void A::operator()(A, A a, int, int a);
void operator new(A, A a, int, int a);
void operator new[](A, A a, int, int a);
void operator delete(A, A a, int, int a);
B A<X, int>::operator new(A, A a, int, int a);
B A::operator delete[](A, A a, int, int a);
operator+(a, b);
operator(a, b);
