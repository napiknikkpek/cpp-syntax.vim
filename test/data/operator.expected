zzzz.aaaaaaaa..t..t.x..zzz..zzz.x..
zzzz.t..aaaaaaaa..t..t.x..zzz..zzz.x..
zzzz.aaaaaaaa...t..t.x..zzz..zzz.x..
zzzz.aaaaaaaa...t..t.x..zzz..zzz.x..
zzzz.t..aaaaaaaa...t..t.x..zzz..zzz.x..
zzzz.aaaaaaaa.aaa.t..t.x..zzz..zzz.x..
zzzz.aaaaaaaa.aaa...t..t.x..zzz..zzz.x..
zzzz.aaaaaaaa.aaaaaa.t..t.x..zzz..zzz.x..
t.t.t..zzz...aaaaaaaa.aaa.t..t.x..zzz..zzz.x..
t.t..aaaaaaaa.aaaaaa...t..t.x..zzz..zzz.x..
aaaaaaaa..x..x..
aaaaaaaa.x..x..
